fabric-client-tags-api-v1/src/client/java/net/fabricmc/fabric/api/tag/client/v1/ClientTags.java

```diff
@@ -46,6 +47,7 @@
  */
 public final class ClientTags {
 	private static final Map<TagKey<?>, Set<Identifier>> LOCAL_TAG_CACHE = new ConcurrentHashMap<>();
+	private static final Map<TagKey<?>, ClientTagsLoader.ChildHolder> LOCAL_TAG_HIERARCHY = new ConcurrentHashMap<>(); //todo use LoadedTag, merge caches?
```

Comment:
`LOCAL_TAG_CACHE` is the `completeIds`, right?

Reply:
Yes, `LOCAL_TAG_CACHE` is the same as `completeIds`